### ISSUE TITLE
More providers

### ISSUE THREAD

**Issue by mrd0ll4r** [jhassine/server-ip-addresses#3]
I stumbled upon [cloudyr/cloudcidrs](https://github.com/cloudyr/cloudcidrs), which has a bunch of additional cloud providers. Unfortunately it's in R, but I think it should be possible to integrate some of it.

They have DigitalOcean (ASN-based), Linode (ASN-based), OVH (ASN-based), Softlayer (ASN- or list-based), Rackspace (ASN-based), Google (DNS-based), in addition to AWS and Azure, with the same methodology as yours.

**Comment by tom10271** [comment]
Agree, it seems this list contains AWS, GCP and Azure only which those trouble makers are all from small and tiny hosting service providers.
